webapp/components/Comment/Comment.vue

```diff
@@ -102,22 +87,25 @@ export default {
     HcCommentForm,
   },
   props: {
-    post: { type: Object, default: () => {} },
-    comment: {
-      type: Object,
-      default() {
-        return {}
-      },
-    },
+    routeHash: { type: String, default: () => '' },
+    post: { type: Object, default: () => ({}) },
+    comment: { type: Object, default: () => ({}) },
     dateTime: { type: [Date, String], default: null },
   },
   computed: {
     ...mapGetters({
       user: 'auth/user',
       isModerator: 'auth/isModerator',
     }),
-    anchor() {
-      return `commentId-${this.comment.id}`
+    isLongComment() {
+      return this.$filters.removeHtml(this.comment.content).length > 180
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-10-16T08:38:50Z" title="Wednesday, October 16th 2019, 10:38:50 am +02:00">Oct 16, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -116,8 +101,15 @@ export default {
       user: 'auth/user',
       isModerator: 'auth/isModerator',
     }),
-    anchor() {
-      return `commentId-${this.comment.id}`
+    isLongComment() {
+      return this.$filters.removeHtml(this.comment.content).length > 180
```


Would love to have this `180` and the next one as the same constant, may be imported.
But will not stop me from approving …